### ISSUE TITLE
Windows 10 TLS/SSL Not Working

### ISSUE THREAD

**Issue by wtf911** [EFTEC/Cyberarms#2]
I'm using Windows 10 Pro 1803 and with the TLS/SSL agent enabled to detect failed remote desktop attempts it isn't picking them up. Could you offer any advice?

**Comment by gabrielbasetti** [comment]
Did you change the windows audit configuration under gpedit.msc ?

![2018-10-02 10_31_48-editor de directivas de grupo local](https://user-images.githubusercontent.com/39503933/46351754-7e9fd780-c62e-11e8-9b42-f754754f807c.png)

**Comment by wtf911** [comment]
Yes I had tried that but it didn't help. I wondered if I was doing
something wrong or it was a recent Windows change. I'm willing to
troubleshoot if there's anything else you'd like me to try.

**Comment by maxemilian** [comment]
You have to ensure you are using NLA for authentication, and close the UDP port for Remote Desktop. 
As UDP can be tampered by spoofing, IDDS does not rely on UDP.
